blocks/story-carousel-block/README.md

```diff
@@ -0,0 +1,44 @@
+# @wpmedia/story-carousel-block
+
+A ANS feed enabled carousel block.
+
+## Props
+
+| **Prop**     | **Required** | **Type**        | **Description**           |
+| ------------ | ------------ | --------------- | ------------------------- |
+| customFields | yes          | PropTypes.shape | Pagebuilder Custom Fields |
+
+### customFields
+
+| **Prop**                       | **Required** | **Type**         | **Description**                                                         |
+| ------------------------------ | ------------ | ---------------- | ----------------------------------------------------------------------- |
+| headerText                     | no           | PropTypes.string | Carousel Header text                                                    |
+| itemHeaderTruncationLines      | no           | PropTypes.string | Number of lines to show in the card header before being truncated.      |
+| itemDescriptionTruncationLines | no           | PropTypes.string | Number of lines to show in the card description before being truncated. |
```

Comment:
can you update the truncated fields based on removing this from the custom field available options 
```suggestion
```

Reply:
Updated